src/test/java/org/fcrepo/migration/handlers/ocfl/ArchiveGroupHandlerTest.java

```diff
@@ -1184,11 +1185,14 @@ private void rawVerifyDoesNotExist(final String ocflObjectId, final String path)
     private ArchiveGroupHandler createHandler(final MigrationType migrationType,
                                               final boolean addExtensions,
                                               final boolean deleteInactive) {
+        // TODO
```

Comment:
Don't leave us hanging here @pwinckles, what needs doing? 😄

Reply:
:)

I needed to add a test and update that code to create a handler that produces atomic resources.

Should be all set now